ngraph/test/visitors/op/logical_and.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, logical_and_op)
+{
+    NodeBuilder::get_ops().register_factory<opset1::LogicalAnd>();
+    auto x1 = make_shared<op::Parameter>(element::boolean, Shape{200});
+    auto x2 = make_shared<op::Parameter>(element::boolean, Shape{200});
+
+    auto auto_broadcast = op::AutoBroadcastType::NUMPY;
+
+    auto logical_and = make_shared<opset1::LogicalAnd>(x1, x2, auto_broadcast);
+    NodeBuilder builder(logical_and);
+    auto g_logical_and = as_type_ptr<opset1::LogicalAnd>(builder.create());
+    const auto expected_attr_count = 1;
+
+    EXPECT_EQ(builder.get_value_map_size(), expected_attr_count);
+    EXPECT_EQ(g_logical_and->get_autob(), logical_and->get_autob());
+}
```

Comment:
Please refactor this test to use [binary_ops.hpp](https://github.com/openvinotoolkit/openvino/blob/a95d59014ccf5324f379dbac680a0587d2c7c566/ngraph/test/visitors/op/binary_ops.hpp) like in this PR #6667

Reply:
done